projects/novo-elements/src/elements/form/NovoFormControl.ts

```diff
@@ -62,6 +61,7 @@ export class NovoFormControl extends FormControl {
   checkboxLabel?: string;
   restrictFieldInteractions?: boolean;
   warning?: string;
+  isHiddenByLogic = false; // if a control is hidden because not releavant (email address in billing profile)
   private historyTimeout: any;
 
```

Comment:
This might not be the way to hide the controls

Reply:
yep, I changed the code accordingly.